Assets/Scripts/Tank/TankService.cs

```diff
@@ -1,37 +1,97 @@
-﻿using UnityEngine;
+﻿using System.Collections.Generic;
+using UnityEngine;
 public class TankService : MonoSingleton<TankService>
 {
-    public TankView tankView;
     private Camera mainCamera;
-    private TankModel tankModel;
-    private TankController tankController;
-    protected Joystick stick;
+
+    [SerializeField]
+    private List<EnemyController> enemyList;
+    [SerializeField]
+    private List<Transform> spawnPositions;
+    [SerializeField]
+    private SpawnerObject spawnerObject;
+    public List<TankScriptableObject> enemiesToGenerate;
+    public Transform enemyParent;
+    public EnemyController redTank;
+    public EnemyController blueTank;
+    public EnemyController greenTank;
+
+    [SerializeField]
+    private TankController playerTank;
+
+    public Transform PlayerTransform{
+        get{
+            Transform playerTransform = playerTank.transform;
+            return playerTransform;
+        }
+    }
+
     private void Start() {
-        GetJoystick();
-        CreateTank();
-        AssignCamera();
+        populateSpawnPositions();
+        CreatePlayer();
+        EnemyController createdEnemy;
```

Comment:
Move to a function called CreateEnemies and use loops for repeated tasks

Reply:
Done